stablehlo/tests/stablehlo_legalize_to_vhlo.0_10_0.mlir

```diff
@@ -2159,6 +2160,13 @@ func.func @type_f8E4M3FNUZ(%arg0: tensor<f8E4M3FNUZ>, %arg1: tensor<f8E4M3FNUZ>)
 }
 // CHECK-LABEL: "type_f8E4M3FNUZ"
 
+func.func @type_f8E4M3B11FNUZ(%arg0: tensor<f8E4M3B11FNUZ>, %arg1: tensor<f8E4M3B11FNUZ>) -> tensor<f8E4M3B11FNUZ> {
```

Comment:
Why is this added?

Reply:
Confirmed that this test passes once this block is removed.


Please regenrate with:

```
stablehlo-translate --serialize --target=0.10.0 --strip-debuginfo stablehlo/tests/stablehlo_legalize_to_vhlo.0_10_0.mlir > stablehlo/tests/stablehlo_legalize_to_vhlo.0_10_0.mlir.bc
```

And update the run commands of all `0_X_0.mlir` files to what they used to be.

Reply:
I'm not sure how this was added either.. Updated.